theme/src/components/variant-select.js

```diff
@@ -68,48 +37,32 @@ function VariantSelect(props) {
   }
 
   variantPages.forEach((match, index) => {
+      let active = false;
       if (match.page.url === path) {
           selectedItem = match;
+          active = true;
       }
-      items.push(
-        <a
-          id={match.variant.shortName}
-          style={{ textDecoration: 'none' }}
-          tabIndex={0}
-          key={match.variant.title}
-          href={match.page.url}
-          onClick={e => anchorClickHandler(e, match.page.url)}
-          onKeyDown={e => onItemEnterKey(e, match.page.url)}
-          onBlur={index === (variantPages.length - 1) ? collapseDropdown : undefined}
-          aria-label={`${match.variant.title}. List items ${index + 1} of ${variantPages.length}`}
-        >
-          <Dropdown.Item>{match.variant.title}</Dropdown.Item>
-        </a>
-      );
+      items.push(<ActionList.Item onKeyDown={(e) => onItemEnterKey(e, match.page.url)} onClick={e => anchorClickHandler(e, match.page.url)} id={match.variant.shortName} key={index} active={active}>
+        <span aria-label={`${match.variant.title}. List items ${index + 1} of ${variantPages.length}`}>
+          {match.variant.title}
+        </span>
+      </ActionList.Item>);
   });
 
-  /** 
-   *  We should use '@primer/react' package as '@primer/components' package depricated and moved to '@primer/react'.
-   *  We have no closing/opening control with current '@primer/components' package, so document.body click event used for closing purpose.
-   */
-  // TODO: We should use 'setOpen' function returned by the useDetails hook when we move to '@primer/react' package. https://primer.style/react/deprecated/Dropdown
-  function onDropDownKeyDown (event) {
-    if (event.key === 'Escape') {
-        document.body.click()
-    }
-  }
 
   return (
-    <div style={{ display: 'flex', flexDirection: 'column' }} ref={wrapper}>
+    <ThemeProvider>
       <label id="label-versions-list-item" htmlFor='variant-select'>Select CLI Version:</label>
-      <Dropdown id="versions-list-item" aria-labelledby={selectedItem?.variant?.shortName} onKeyDown={onDropDownKeyDown} overlay={props.overlay}>
```

Comment:
With the latest changes, VoiceOver on Mac does not announce the selected version (before clicking on the menu button). Can we fix it ?

Reply:
@prabum3 I added title for selectedItem to aria-label.

Reply:
It would be better to add aria-label to `ActionMenu` instead of `ActionMenu.Button`. Now the screen reader is announcing it twice when selected.

Reply:
@prabum3 I added condition for exposing `aria-label` only if `ActionMenu` closed. If add `aria-label` to ActionMenu it could be announce twice also.